service/host.go

```diff
@@ -60,14 +60,17 @@ func (uis *UIServer) hostPage(w http.ResponseWriter, r *http.Request) {
 		uis.LoggedError(w, r, http.StatusInternalServerError, err)
 		return
 	}
-	runningTask := &task.Task{}
+	var runningTask *task.Task
 	if h.RunningTask != "" {
 		runningTask, err = task.FindOne(task.ById(h.RunningTask))
 		if err != nil {
 			uis.LoggedError(w, r, http.StatusInternalServerError, err)
 			return
 		}
 	}
+	if runningTask == nil {
```

Comment:
why is it happening that the running task doesn't exist in the database?

Reply:
It's hard to tell (since the task isn't in the database), but I think it may have been a task I deleted by hand when we were recovering from an outage.

Reply:
maybe this case should also be caught on 67 and return a 500 code?

Reply:
I had initially thought it would be better to return the host anyway, but I think the data inconsistency is rare enough that the approach you suggested is better. Fixed.